apiserver/application/application.go

```diff
@@ -339,6 +354,71 @@ func (api *API) Update(args params.ApplicationUpdate) error {
 	return nil
 }
 
+// UpdateApplicationSeries updates the application series.  Series for
```

Comment:
We don't use double spaces after a full stop. There's at least a couple of places.